build.gradle

```diff
@@ -69,6 +69,16 @@ dependencies {
 
     // Java Doc in XML Format
     xmlDoclet 'com.manticore-projects.tools:xml-doclet:+'
+    
```

Comment:
I am familiar with Jackson and use it myself for serialization.
But I don't like it here because it depends a massive 3rd party dependency for rather limited benefit:

1) the serialization is for caching only
2) the meta data are fairly simple and not nested

So I would like to ask for either writing the JSON manually (if you really need/insist in JSON serialisation) or just your the standard Java serialisation (since its for caching only)

Reply:
Well, this is one of the core extensions of existing functionality for my usecase (and I believe others would benefit from it too). Writing the serialization as a series of append() is possible, but I consider that to be an ugly way of doing the serialization - not mentioning the de-serialization back. I could implement it as an external class, but then would need additional public methods added (or change too many members to be public - which, btw, the public visibility widely used my not be the best thing).